src/isc-dhcp/isc-dhcp-4.3.1_dhcrelay-custom-circuit_id-remote_id-and-bridge-iface-support.patch

```diff
@@ -71,7 +97,81 @@ diff -ruN a/isc-dhcp/relay/dhcrelay.c b/isc-dhcp/relay/dhcrelay.c
  			add_agent_options = 1;
  		} else if (!strcmp(argv[i], "-A")) {
  #ifdef DHCPv6
-@@ -937,6 +959,166 @@
+@@ -454,6 +488,10 @@
+ 	} else 
+ 		log_perror = 0;
+ 
++	/* Load our interface name-alias map */
++	if (load_interface_alias_map() != 0)
++		log_fatal("Failed to load interface name-alias map.");
++
+ 	/* Set default port */
+ 	if (local_family == AF_INET) {
+  		service_local = "bootps";
+@@ -602,6 +640,8 @@
+ 	dispatch();
+ 
+ 	/* In fact dispatch() never returns. */
++	free_interface_alias_map();
++
+ 	return (0);
+ }
+ 
+@@ -690,10 +730,9 @@
+ 				&to, htop) < 0) {
+ 			++server_packet_errors;
+ 		} else {
+-			log_debug("Forwarded BOOTREPLY for %s to %s",
+-			       print_hw_addr(packet->htype, packet->hlen,
+-					      packet->chaddr),
+-			       inet_ntoa(to.sin_addr));
++			//log_debug("Forwarded BOOTREPLY for %s to %s",
++			//          print_hw_addr(packet->htype, packet->hlen, packet->chaddr),
++			//          inet_ntoa(to.sin_addr));
```

Comment:
remove the commented lines?

Reply:
I would have liked to leave them uncommented, but unfortunately, the behavior of `log_debug()` is to print to stdout, and we have configured supervisord to forward stdout messages to the syslog, so leaving these `log_debug()` calls in spams the syslog.

I considered removing these altogether, but they may come in handy for future debugging, so I thought leaving them commented out is better for posterity's sake than having to rewrite them again, even if it leaves a bit of clutter.

If you still prefer I remove them, I will. Just let me know.